<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5DishPart2</class>
 <widget class="QWidget" name="CE5DishPart2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leDeptName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>part_name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ADG code</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="leDept">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_part</string>
     </property>
     <property name="Type" stdset="0">
      <string>1</string>
     </property>
     <property name="First" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="C5LineEditWithSelector" name="lineEdit_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_adgcode</string>
     </property>
     <property name="Type" stdset="0">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="C5LineEditWithSelector" name="leColor">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_color</string>
     </property>
     <property name="SetColor" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Queue</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="C5LineEditWithSelector" name="lineEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="Primary" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="C5LineEditWithSelector" name="lineEdit_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_queue</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="C5LineEditWithSelector" name="lineEdit_2">
     <property name="Field" stdset="0">
      <string>f_name</string>
     </property>
     <property name="Type" stdset="0">
      <string>0</string>
     </property>
     <property name="Check" stdset="0">
      <string>empty=&quot;Name&quot;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
